CHANGELOG.md

```diff
@@ -1,14 +1,15 @@
-# CRPropa vNext
+## CRPropa vNext
 
 ### Bug fixes:
 * Re-added ToroidalHaloField and LogarithmicSpiralField models. Note, that the class name was also corrected in spelling: TorroidalHaloField --> ToroidalHaloField
 * Synchronized signature of ParticleSplitting constructor
 
 ### New features:
-* new candidate property tagOrigin to trace back which source or which interaction created the candidate
-* new interace for massdistributions given on a Grid1f
-* grids can be restricted to the volume without repetition
-* sourceFeature to sample the source position from a given massdistribution
+* New candidate property tagOrigin to trace back which source or which interaction created the candidate
+* New interace for massdistributions given on a Grid1f
+* Grids can be restricted to the volume without repetition
+* SourceFeature to sample the source position from a given massdistribution
+* Added default parameter for clipVolume in GridProperties.
```

Comment:
I don't think its necessary to mention here the default value, as `clipVolume` is already a new feature.

Reply:
I merged the two comments on the clipVolume functionality